rosetta/rosetta/projects/pax/README.md

```diff
@@ -113,6 +113,22 @@ ENABLE_FP8=1 bash paxml/contrib/gpu/scripts_gpu/run_pile_singlenode.sh /opt/paxm
 
 Note that packing is currently not supported when using TE. All configs disable packing by default, but beware that if packing is manually enabled, training with TE will error.
 
+### Native FP8 Support
+The most recent version of XLA already supports FP8 GEMMs through the utilization of custom quantization operations provided by Paxml/Praxis. To enable this feature, users simply need to include the option `--fdl.USE_FP8=True` in their experiment configuration. This will activate the recommended layers to use FP8 GEMMs within the transformer layer. It's important to note that `ENABLE_TE` must be turned off.
```

Comment:
Just as a writing style suggestion, please use the full terminology, e.g. general matrix multiplication (GEMM), so that users can get the context more easily.

Reply:
Thanks for the suggestion. Done.